types/grecaptcha/index.d.ts

```diff
@@ -1,9 +1,10 @@
-// Type definitions for Google Recaptcha 2.0
+// Type definitions for Google Recaptcha 3.0
 // Project: https://www.google.com/recaptcha
 // Definitions by: Kristof Mattei <http://kristofmattei.be>
 //                 Martin Costello <https://martincostello.com/>
 //                 Ruslan Arkhipau <https://github.com/DethAriel>
 //                 Rafael Tavares <https://github.com/rafaeltavares>
+//                 Florian Rohrer <https://github.com/RohrerF>
 // Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
 
 declare var grecaptcha: ReCaptchaV2.ReCaptcha;
```

Comment:
should this be updated as well?

Reply:
I would avoid renaming it that at this moment.